extensions/wikia/AdEngine/js/video/player/porvata/porvataPlayerFactory.js

```diff
@@ -56,6 +61,9 @@ define('ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory', ['wikia.l
 				ima.getAdsManager().resume();
 			},
 			setVolume: function (volume) {
+				if (videoAd) {
+					videoAd.muted = volume === 0;
+				}
 				return ima.getAdsManager().setVolume(volume);
```

Comment:
Built-in `ima.setVolume(0)` method doesn't set `muted` attribute on video tag by default.
So if we set some volume without disabling mute flag it won't be unmuted automatically.